### PR TITLE
[xlang] Hotly fix the importing error for python call Java.

### PR DESCRIPTION
**Pull request by jovany-wang**
## Why are these changes needed?
The PR https://github.com/ray-project/ray/pull/22820 introduced a API breakage for xlang usage, causing that `ray.java_actor_class` has not been available any longer from then on.

I'm fixing it in this PR. We should remove these top level APIs in 2.0 instead of minor versions.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by jovany-wang** [comment]
@ericl Is there any place that we can track the API changes for 2.0?

File: python/ray/__init__.py [hunk 1]
```diff
@@ -155,6 +155,7 @@ def _configure_system():
 # some functions in the worker.
 import ray.actor  # noqa: E402,F401
 from ray.actor import method  # noqa: E402
+from ray.cross_language import java_function, java_actor_class  # noqa: E402
```
Comment (SongGuyang) [inline python/ray/__init__.py:158]: Can you add a `TODO` here which indicates that we should remove this in 2.0?

Comment (jovany-wang) [reply python/ray/__init__.py:158]: Let us see if there is a tracker for 2.0 API changes. If yes, it's no need to add TODO right?

Comment (jovany-wang) [reply python/ray/__init__.py:159]: @SongGuyang  I added a todo comment and left it in my private TODO list.

**Review by SongGuyang — APPROVED** [review]

**Pull request merged**